modal/runner/engines/vllm.py

```diff
@@ -77,11 +77,9 @@ async def generate(self, payload: CompletionPayload, params):
         try:
             import time
 
-            tags = {"model": self.engine_args.model}
-            with timer("engine.generate()", tags=tags):
```

Comment:
we're removing this bc it was instant right?

Reply:
yep! exactly, just cleaning up